sys/kern/subr_vmem.c

```diff
@@ -1702,7 +1702,7 @@ DB_SHOW_COMMAND(vmemdump, vmemdump)
 		return;
 	}
 
-	vmem_dump((const vmem_t *)addr, db_printf);
+	vmem_dump(DB_DATA_PTR(addr, vmem_t), db_printf);
```

Comment:
const? (and below)

Reply:
Oh, yes, I switched to keeping the const part way in and forgot to go back and fix the few from before.